menas/src/main/scala/za/co/absa/enceladus/menas/WebSecurityConfig.scala

```diff
@@ -43,6 +43,11 @@ import javax.servlet.http.HttpServletResponse
 import za.co.absa.enceladus.menas.auth.InMemoryMenasAuthentication
 import za.co.absa.enceladus.menas.auth.KerberosMenasAuthentication
 import za.co.absa.enceladus.menas.auth.MenasAuthentication
+import org.springframework.security.authentication.AuthenticationManager
+import org.springframework.security.kerberos.web.authentication.SpnegoAuthenticationProcessingFilter
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter
+import org.springframework.security.kerberos.web.authentication.SpnegoEntryPoint
+import org.springframework.security.web.authentication.www.BasicAuthenticationFilter
```

Comment:
imports 31, 32, 46, 47 and 50 are not being used anymore. Not sure if all by your edit, or for a long time, but still.